lib/child_process.js

```diff
@@ -277,7 +271,7 @@ exports.execFile = function(file /*, args, options, callback*/) {
         if (!encoding)
           _stderr.push(chunk);
         else
-          _stderr += chunk;
+          _stderr += chunk.toString(encoding);
```

Comment:
Yes, this is true. That's why `setEncoding()` was being used initially. This will need to be solved differently I think.